vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -136,8 +136,10 @@ def parse_container_definition(container_def, pod_id)
         :cpu_cores         => container_def["cpu"].to_f / 1000
       }
       ports = container_def["ports"]
-      new_result[:container_port_configs] = ports.collect do |port_entry|
-        parse_container_port_config(port_entry, pod_id, container_def["name"])
+      unless ports.nil?
```

Comment:
Oh wait, I am going back on that...if the ports is nil, we actually want the key set to an empty array.  This will allow the refresher to remove any existing records in the inventory. So, even the `if ports.present?` is wrong.

I think this is more accurate as

```ruby
new_result[:container_port_configs] = ports.to_miq_a.collect do |port_entry|
  parse_container_port_config(port_entry, pod_id, container_def["name"])
end
```